build.gradle

```diff
@@ -352,7 +352,7 @@ task uploadAll(type: UploadS3Task) {
   bucket 'prelert-artifacts'
   // Upload ALL artifacts (including the dependency report) in this task
   def fileDir = fileTree("${buildDir}/distributions").matching {
-    include "ml-cpp-${project.version}*.zip", "dependencies-${version}.csv"
+    include "ml-cpp-*${project.version}*.zip", "dependencies-${version}.csv"
```

Comment:
In #2464 the pattern was `ml-cpp*${project.version}*.zip` instead of `ml-cpp-*${project.version}*.zip`. It would be nice to use exactly the same pattern in both places, to avoid confusion in the future.